### ISSUE TITLE
Bug Undefined variable: "$navbar-item" in sass/components/navbar.sass

### ISSUE THREAD

**Issue by bstashio** [jgthms/bulma#1168]
### Overview of the problem

This is about the Bulma **CSS framework**

I'm using Bulma **version** [0.5.1]

### Description

When compiling using `sass bulma.sass bulma.css` we get the following error:

```
Error: Undefined variable: "$navbar-item".
        on line 61 of sass/components/navbar.sass, in `@content'
        from line 209 of sass/utilities/mixins.sass, in `desktop'
        from line 55 of sass/components/navbar.sass
        from line 12 of sass/components/_all.sass
        from line 6 of bulma.sass
```

### Solution

It's obvious that the variable `$navbar-item` is not defined and was mistakenly used to set the `color` property in `navbar.sass`, I think it should be changed to `$color` instead.  The code on line 61  of sass/components/navbar.sass would become as follows:

```
color: $color
```

**Comment by mkolassa** [comment]
The fix applied by @jgthms in #1168 did not include the fix provided by @bstashio.

**Comment by mkolassa** [comment]
My apologies, I did not realize that there was a delay in merging the fix into master.
